q2_feature_table/tests/test_merge.py

```diff
@@ -93,6 +93,52 @@ def test_invalid_overlap_method(self):
         with self.assertRaisesRegex(ValueError, 'overlap method'):
             merge(t1, t2, 'peanut')
 
+    def test_sum_full_overlap(self):
```

Comment:
Could you add a `test_sum_partial_overlap`, where some but not all of the sample ids _and_ some but not all of the features ids overlap? I don't expect any issues, but I think that's a very common use case in practice, so it makes sense to have a test for it.

Reply:
Good catch @gregcaporaso - addressed in a [squash commit](https://github.com/qiime2/q2-feature-table/pull/142/commits/8684501f26c59c24076f150f581b10981cc6a927) 🎃 .

Reply:
👍